src/predicates.ts

```diff
@@ -3,50 +3,54 @@ import { pause } from './small';
 import { Predicate, waitFor } from './wait-for';
 import * as pFilter from 'p-filter';
 
-export const isElementFound = async (
+const checkVisibility = async (
```

Comment:
let's put it inside `isElementVisible` as `const` variable with naming `isVisible` or smth